### PR TITLE
Add ref code mapping

### LINKED ISSUE #163: [Deliveroo & Uber Eats] Manual ref code mapping
We need to explain how to enter the ref codes in these 2 platforms (manual mapping)

@atorin Do you have access to test or customer accounts on the back-office of both portals?

Maybe we should create a "Map Ref Codes" section to explain the process. Let me know if you have a better idea.

### PR DESCRIPTION
**Pull request by atorin**
- Procedure for Deliveroo
- Procedure for Uber Eats
- Move FAQs page

Closes #163

### CODE REVIEW

**Comment by antmonnier** [comment]
Great, I think we just need to include a link to the new page in the "Connect to HubRise" page. We should also include a link to the "Configuration" page.

My idea is that we can use "Connect to HubRise" as an entry point:
1. Connect the bridge
2. Enter location ID / store UUID
3. Configure the bridge
4. Map ref codes
5. Place a test order (?)

File: content/apps/deliveroo/en/connect-hubrise.md [hunk 1]
```diff
@@ -43,3 +43,20 @@ Now that you have connected Deliveroo Bridge to your location on HubRise, a new
 ---
 
 Once you are finished, click **Save** to complete the connection process.
+
+## Configure the Deliveroo Bridge
```
Comment (atorin) [inline content/apps/deliveroo/en/connect-hubrise.md:47]: Alternatively, these last three sections could become subsections of a "Next Steps" (or similar) section.

**Comment by atorin** [comment]
I've added those additional points.

File: content/apps/deliveroo/en/connect-hubrise.md (hunk 1 above)
Comment (antmonnier) [reply content/apps/deliveroo/en/connect-hubrise.md:47]: I personally like the way it is currently structured, one simple step at a time

**Pull request merged**